finalPrep/modZeroHero.js

```diff
@@ -1,40 +1,70 @@
-// Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
-
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = 'man spider';
+var specialAbility = 'being absolutely terrifying';
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
-//   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+//   catchphrase should be assigned to a string that uses interpolation to
+//include the specialAbility
 
+var greeting = 'scared yet?' + heroName
+var catchphrase = `just try to stomp me! ${specialAbility}`
+console.log(greeting)
+console.log(catchphrase)
 // Declare two variables - power AND energy - set to integers
+var power = 100;
+var energy = 85;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = 500 * power;
+var fullEnergy = 150 + energy;
 
-// Declare two variables - isHuman and identityConcealed - assigned to booleans
+console.log(fullPower)
+console.log(fullEnergy)
 
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = false;
+var identityConcealed =false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["test1", "test2", "test3"]
+var sidekicks = ["example1", "example2", "example3"]
 // Print the first sidekick to your console
+console.log(sidekicks[0])
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2])
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("newArchEnemy")
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies)
 
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift()
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks)
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  if(dangerLevel >= 10 && dangerLevel <= 50) {
```

Comment:
You can leverage the order you place these conditionals to simplify things here. What you have is fine, but consider making your fist if statement your else statement instead.

Reply:
Ahhhh. I _think_ I get it. That does indeed make more sense.